packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/palette/PalettePortal.tsx

```diff
@@ -12,17 +12,14 @@
  *******************************************************************************/
 
 import { createPortal } from 'react-dom';
-import { ReactFlowState, useStore } from 'reactflow';
 import { PalettePortalProps } from './PalettePortal.types';
 
-const selector = (state: ReactFlowState) => state.domNode?.querySelector('.react-flow__renderer');
-
 export const PalettePortal = ({ children }: PalettePortalProps) => {
-  const wrapperRef = useStore(selector);
+  const wrapperRef = document.querySelector('.react-flow');
 
   if (!wrapperRef) {
     return null;
   }
 
-  return createPortal(children, wrapperRef);
+  return createPortal(<div style={{ zIndex: 5, position: 'absolute' }}>{children}</div>, wrapperRef);
```

Comment:
Could you add a comment here please explaining why you have to put a zIndex with 5 as value? Thank you

Reply:
done